programs/helium-entity-manager/src/instructions/issue_data_only_entity_v0.rs

```diff
@@ -0,0 +1,215 @@
+    seeds::program = token_metadata_program.key(),
+    bump,
+  )]
+  pub collection_master_edition: UncheckedAccount<'info>,
+  #[account(
+    mut,
+    seeds = ["data_only_config".as_bytes(), dao.key().as_ref()],
+    bump,
+    has_one = collection,
+    has_one = merkle_tree,
+  )]
+  pub data_only_config: Box<Account<'info, DataOnlyConfigV0>>,
+  /// CHECK: Signs as a verified creator to make searching easier
+  #[account(
+    seeds = ["entity_creator".as_bytes(), dao.key().as_ref()],
+    bump,
+  )]
+  pub entity_creator: UncheckedAccount<'info>,
+  pub dao: Box<Account<'info, DaoV0>>,
+  #[account(
+    init,
+    payer = payer,
+    space = 8 + std::mem::size_of::<KeyToAssetV0>() + 1 + args.entity_key.len(),
+    seeds = [
+      "key_to_asset".as_bytes(),
+      dao.key().as_ref(),
+      &hash(&args.entity_key[..]).to_bytes()
+    ],
+    bump
+  )]
+  pub key_to_asset: Box<Account<'info, KeyToAssetV0>>,
+  #[account(
+      mut,
+      seeds = [merkle_tree.key().as_ref()],
+      seeds::program = bubblegum_program.key(),
+      bump,
+  )]
+  pub tree_authority: Box<Account<'info, TreeConfig>>,
+  /// CHECK: Used in cpi
+  pub recipient: AccountInfo<'info>,
+  /// CHECK: Used in cpi
+  #[account(mut)]
+  pub merkle_tree: AccountInfo<'info>,
+
+  /// CHECK: checked with seeds
+  #[account(mut,
+    seeds = [
+      "data_only_escrow".as_bytes(),
+      data_only_config.key().as_ref(),
+    ],
+    bump
+  )]
+  pub data_only_escrow: AccountInfo<'info>,
+  #[account(
+    seeds = ["collection_cpi".as_bytes()],
+    seeds::program = bubblegum_program.key(),
+    bump,
+  )]
+  /// CHECK: Used in cpi
+  pub bubblegum_signer: UncheckedAccount<'info>,
+
+  /// CHECK: Verified by constraint  
+  #[account(address = mpl_token_metadata::ID)]
+  pub token_metadata_program: AccountInfo<'info>,
+  pub log_wrapper: Program<'info, Noop>,
+  pub bubblegum_program: Program<'info, Bubblegum>,
+  pub compression_program: Program<'info, SplAccountCompression>,
+  pub system_program: Program<'info, System>,
+}
+
+impl<'info> IssueDataOnlyEntityV0<'info> {
+  fn mint_to_collection_ctx(&self) -> CpiContext<'_, '_, '_, 'info, MintToCollectionV1<'info>> {
+    let cpi_accounts = MintToCollectionV1 {
+      tree_authority: self.tree_authority.to_account_info(),
+      leaf_delegate: self.recipient.to_account_info(),
+      leaf_owner: self.recipient.to_account_info(),
+      merkle_tree: self.merkle_tree.to_account_info(),
+      payer: self.payer.to_account_info(),
+      tree_delegate: self.data_only_config.to_account_info(),
+      log_wrapper: self.log_wrapper.to_account_info(),
+      compression_program: self.compression_program.to_account_info(),
+      system_program: self.system_program.to_account_info(),
+      collection_authority: self.data_only_config.to_account_info(),
+      collection_authority_record_pda: self.bubblegum_program.to_account_info(),
```

Comment:
This naming throws me off, im always like why is a pda assigned to a program.
Is this actually `collection_record_pda_authority`?

Reply:
From the MintToCollectionV1 struct:
```
/// CHECK: Optional collection authority record PDA.
/// If there is no collecton authority record PDA then
/// this must be the Bubblegum program address.
```